assets/css/styles.scss

```diff
@@ -305,6 +305,21 @@ $cart-heading-line-height: ($navbar-height - ($panel-heading-padding-vertical *
   }
 }
 
+/* fix too long list order*/
+.sticky-inner-wrapper {
+ overflow-y: auto; 
+ height: 85%; 
+}
+
+.panel.panel-default {
```

Comment:
This selector targets too much elements! It should be more restrictive.
Moreover, I think it doesn't change anything.